activity_calendar/templatetags/calendar_tags.py

```diff
@@ -39,6 +39,11 @@ def get_opening_time(context, filter=None):
     return open_date
 
 
+@register.filter
+def is_subscribed_to(user, activity_moment):
+    return activity_moment.get_user_subscriptions(user)
```

Comment:
Judging from the method name, shouldn't this return a boolean?

Reply:
Technically yes, this hasn't caught my attention yet as the template engine interprets an empty iterable as False XD